### ISSUE TITLE
yolov5s showing 56 mb size

### ISSUE THREAD

**Issue by nralka2007** [ultralytics/yolov5#5039]
## ❔Question

I am training a yolov5s  model on custom dataset. I am noticing that the size of the file is 56 mb when it should be 14 mb. What is the reasom. I am using default value for training as shown 
![best_pt_size](https://user-images.githubusercontent.com/26127036/135800903-eecbc5d8-03f4-48d5-96da-776a91c13dee.png)
![train_params](https://user-images.githubusercontent.com/26127036/135800938-db9ade1d-490a-46c7-9015-1600bbcb7fe2.png)

My train command is like this:
 python train.py  --batch 16 --epochs 400 --data './data.yaml' --cfg ./models/custom_yolov5s.yaml --weights './weights/yolov5s.pt'

custome_yolov5s.yaml is same as yolov5s.yaml, just that nc is set to 2 instead of 80

**Comment by glenn-jocher** [comment]
@nralka2007 YOLOv5 🚀 training checkpoints should be 4X the size of final trained checkpoints, as they carry not just a FP16 model, but a FP16 [EMA](https://www.tensorflow.org/api_docs/python/tf/train/ExponentialMovingAverage) and an FP32 optimizer of the same size as the model (each model parameter has its own FP32 gradient saved within the optimizer). Final checkpoints contain only an FP16 model, with the EMA and optimizer both stripped after the final epoch of training:
https://github.com/ultralytics/yolov5/blob/75feeb797c4a9553f4274860c6c278f1fc628f60/train.py#L435-L439

The `strip_optimizer()` function updates the checkpoint dictionary value for `model`, replacing it with `ema`, and sets `ema` and `optimizer` keys to `None`, which will reduce the checkpoint size by 3/4.
https://github.com/ultralytics/yolov5/blob/75feeb797c4a9553f4274860c6c278f1fc628f60/utils/general.py#L495-L509

Final trained checkpoints after `strip_optimizer()` should match the README table sizes below.
<p align="center"><img width="700" alt="YOLOv5 Models" src="https://github.com/ultralytics/yolov5/releases/download/v1.0/model_comparison.png"></p>
